### PR TITLE
Build command: update Xcode version validation for Xcode 10+ (#351)

### PR DESCRIPTION
**Pull request by thzinc**
I did not see a contributor's guide, so please let me know if there are any changes needed to this work. 

Thanks!

### CODE REVIEW

**Comment by ViachelsavGerasimenok** [comment]
Hi All, when this Pull Request will merge?

File: lib/shenzhen/commands/build.rb [hunk 1]
```diff
@@ -150,7 +150,8 @@
 
   def validate_xcode_version!
     version = Shenzhen::XcodeBuild.version
-    say_error "Shenzhen requires Xcode 4 (found #{version}). Please install or switch to the latest Xcode." and abort if version < "4.0.0"
+    majorVersion = version.split('.')[0].to_i
```
Comment (dankimio) [inline lib/shenzhen/commands/build.rb]: Can you fix the naming? It must be consistent with the project.

Comment (thzinc) [reply lib/shenzhen/commands/build.rb]: Doh, sure thing.

Comment (thzinc) [reply lib/shenzhen/commands/build.rb]: I've made the requested change. Thanks for your feedback!

**Comment by dankimio** [comment]
@thzinc Thanks for the PR!

**Pull request merged**